src/modules/mahjong/DeadPile/DeadPile.ts

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
these are mahjong related changes, seems like you didn't rebase you branch?

Reply:
Oh... Probably messed it up. Looks like I merged